vendor/modules.txt

```diff
@@ -109,8 +109,8 @@ github.com/karrick/godirwalk
 github.com/kballard/go-shellquote
 # github.com/kr/pretty v0.3.0
 ## explicit; go 1.12
-# github.com/lanrat/extsort v1.0.2
-## explicit; go 1.13
+# github.com/lanrat/extsort v1.4.2
+## explicit; go 1.23
```

Comment:
Unfortunately, looks like the Go version in extsort was bumped in version v1.2.0. s5cmd would need to also update Go to 1.23 to take in this change